cmd/kubeadm/app/apis/kubeadm/v1beta4/types.go

```diff
@@ -508,3 +508,8 @@ type Arg struct {
 	Name  string `json:"name"`
 	Value string `json:"value"`
 }
+
+// EnvVar represents an environment variable present in a Container.
+type EnvVar struct {
+	corev1.EnvVar `json:",inline"`
+}
```

Comment:
it didn't work with type alias?

`type EnvVa rcorev1.EnvVar`

Reply:
Either way is ok, but this is more convenient for later expansion if we need it.

Reply:
ack, SGTM.